documentation/modules/ROOT/examples/orgs_vars/superadmin/env/dev/controller_credentials.d/controller_credentials_scm.yml

```diff
@@ -5,9 +5,7 @@ controller_credentials:
     credential_type: "Source Control"
     organization: "{{ orgs }}"
     inputs:
-      username: "<CHANGE_ME>"
+      username: "${GITLAB_USER}"
       ssh_key_data: |
-        -----BEGIN OPENSSH PRIVATE KEY-----
-        XXXXXXXXX <CHANGE_ME> XXXXXXXXX
-        -----END OPENSSH PRIVATE KEY-----
+${GITLAB_PRIV_KEY}
```

Comment:
I'd prefer not changing this, as it looks clearer as it was originally written.

Reply:
ok. we can leave it as it was